environments/ithc/backend_lb_config.yaml

```diff
@@ -290,3 +290,6 @@ gateways:
         component: hmi-inbound-adapter
       - product: hmc
         component: hmi-outbound-adapter
+    #SSCS
```

Comment:
Please append to https://github.com/hmcts/azure-platform-terraform/blob/feature/sscs-10742-add-hearings-api-lb/environments/ithc/backend_lb_config.yaml#L142